cmd/dbgenerator/main.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	dbm "github.com/cometbft/cometbft-db"
+	"github.com/cosmos/iavl"
+)
+
+const (
+	DefaultCacheSize = 1000
+)
+
+func main() {
+	args := os.Args[1:]
+	if len(args) < 4 {
+		fmt.Fprintln(os.Stderr, "Usage: dbgenerator <dbtype> <dbdir> <random|sequential> <version>")
```

Comment:
Can we get a description for what these are / a readme? (Maybe chatGPT can help?)

Quite cool that you made a tool to help testing / generate sample trees :)